src/main/resources/db/migration/R__Comments.sql

```diff
@@ -541,7 +541,6 @@ COMMENT ON TABLE accelerator.user_deliverable_categories IS 'Which deliverable c
 
 COMMENT ON TABLE accelerator.vote_options IS '(Enum) Available vote options.';
 
-
```

Comment:
The double newline here was to visually separate the sections for different schemas a little more. If we don't want that, we should remove the extra lines between other schemas too.